code/Gamemodes/Modes/FFA/FreeForAll.cs

```diff
@@ -21,6 +21,9 @@ public partial class FreeForAll : Gamemode
 	/// </summary>
 	public Task NextTurnTask { get; set; }
 
+	private Queue<Player> LateJoinSpawnQueue { get; set; } = new();
+	private string[] _lateJoinPhrases = new string[3] { "arrived fashionably late!", "has dropped in unexpectedly!", "finally decided to showed up!" };

```

Comment:
This is fine for now but we'll have to refactor it later IMO.

Reply:
Do you mean that the use of the queue will need to be refactored, or the strings?

Reply:
The strings, in the future, we will have some kind of map or maybe even a strings file (tyler mcv reference)

Reply:
Yeah, I agree!! I just needed somewhere to put them for now. I will dump them in the strings file un-encrypted just for him...